assets/css/experience.css

```diff
@@ -526,24 +531,26 @@ ul ol{
   background: transparent;
 } */
 
- .content > * + * {
+ /* .content > * + * {
   margin-top: 1rem;
-} 
+}  */
 
 .volunteerTitle {
   font-size: 1.5rem;
   line-height: 1.2;
-  margin-top: 0.5em;
+  /* margin-top: 0.5em; */
```

Comment:
Delete the unused code please

Reply:
Remove all the commented out code. The PR is ready for merge after that

Reply:
remove all the commented codes or the code which i commented? @smaranjitghose

Reply:
All the commented codes please

Reply:
Done @smaranjitghose